### PR TITLE
CAS-85 - Proxy Analytics

### PR DESCRIPTION
**Pull request by germanurrus**
- updates on Plausible script

### CODE REVIEW

File: frontend/packages/client/public/index.html [hunk 1]
```diff
@@ -28,8 +28,9 @@
     <title>CAST</title>
     <script
       defer
+      data-api="/api/event"
       data-domain="cast.fyi"
-      src="https://plausible.io/js/plausible.hash.js"
+      src="https://data.brud.dev/js/script.js"
```
Comment (dbslone) [inline frontend/packages/client/public/index.html:33]: Wouldn't this only use our dev environment based on the url provided here? Is plausible being included as a package somewhere? Not sure I understand the change being made here.

Comment (germanurrus) [reply frontend/packages/client/public/index.html:33]: My bad, I didn't add any explanation about this change. This is related to this: https://plausible.io/docs/proxy/introduction and could be possible that our plausible integration gets blocked by adblockers. 
At the end, the scripts running are the same it only changes the location from where we are pulling them from to run and make the adblocker "believe" that is one or our own scripts running.

Comment (dbslone) [reply frontend/packages/client/public/index.html:33]: I think we should do this on the prod domain so it's `data.cast.fyi/script.js`. Can you also work with @fersan1985 to set up the DNS.

Comment (fersan1985) [reply frontend/packages/client/public/index.html:33]: Hey @germanurrus or @dbslone That data.brud.dev where does it point to? Our brud.dev is a wildcard that we use in our ingress to point traffic to a specific service.

**Review by rvmelkonian — APPROVED** [review]

**Comment by germanurrus** [comment]
Changes where deployed to staging directly https://github.com/DapperCollectives/CAST/pull/191

**Pull request closed without merging**